File: Data Structures & Algorithms/concatenation-of-array/submission-1.kt
class Solution {
    fun getConcatenation(nums: IntArray): IntArray {
        val n = nums.size
        val ans = Array<Int>(size=2*n){0}

        for(i in 0 .. nums.size-1){
            ans[i]=nums[i]
            ans[i+n]=nums[i]
        }

return ans.toIntArray()
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        ans = []
        res = {}
        for i in nums:
            res[i] = res.get(i,0) + 1
        print (res)

        sorted_by_val = dict(sorted(res.items(), key=lambda item: item[1],reverse=True))
        print (sorted_by_val)
        ans =  list(sorted_by_val)[:k]
        return ans
        